src/main/java/mate/academy/bookstore/BookstoreApplication.java

```diff
@@ -1,29 +1,27 @@
 package mate.academy.bookstore;
 
 import java.math.BigDecimal;
-import mate.academy.bookstore.model.Book;
+import lombok.RequiredArgsConstructor;
+import mate.academy.bookstore.dto.CreateBookRequestDto;
 import mate.academy.bookstore.service.BookService;
 import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.context.annotation.Bean;
 
+@RequiredArgsConstructor
 @SpringBootApplication
 public class BookstoreApplication {
     private final BookService bookService;
 
-    public BookstoreApplication(BookService bookService) {
-        this.bookService = bookService;
-    }
-
     public static void main(String[] args) {
         SpringApplication.run(BookstoreApplication.class, args);
     }
 
```

Comment:
please read task description, you have to remove this bean, because you don't need to do something in main

Reply:
I mean CommandLineRunner bean